### ISSUE TITLE
NVDA treats definition lists as unordered lists in HTML documents

### ISSUE THREAD

**Issue by vkolp** [nvaccess/nvda#15351]
### Steps to reproduce:
1. Open Chrome.
2. Go to https://www.w3.org/WAI/WCAG21/Understanding/non-text-content
3. In browse mode, navigate to the main content of the page, then navigate to the 1st list.  
4. Inspect the list element in Chrome Devtools and verify that this is not an unordered list but a definition list.
### Actual behavior:

NVDA announces this list as if it was an unordered list. When navigating to the list, it reads: "list  with 12 items  Controls, Input".
### Expected behavior:

Since this list is not an unordered list, but a definition list, NVDA should treat it as such. In this case, it should announce something like: "definition list of 6 items", and then the 1st term "Controls, input" should be announced.    
### NVDA logs, crash dumps and other attachments:
N/A
#### NVDA installed/portable/running from source:
Installed
#### NVDA version:
2023.2rc1
#### Windows version:
Windows 11 22H2 (OS Build 22621.2134)
#### Name and version of other software in use when reproducing the issue:
Chrome 116
#### Does the issue still occur after restarting your computer?
Yes
#### Have you tried any other versions of NVDA? If so, please report their behaviors.
Yes. Reproducible with NVDA 2023.1, NVDA 2022x, and so on.
#### If NVDA add-ons are disabled, is your problem still occurring?
Yes
#### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?
Yes

**Comment by seanbudd** [comment]
Closing as duplicate of #3858
